scalalib/src/mill/scalalib/ScalaModule.scala

```diff
@@ -549,6 +549,14 @@ trait ScalaModule extends JavaModule with TestModule.ScalaModuleBase { outer =>
     else ZincWorkerUtil.scalaBinaryVersion(scalaVersion())
   }
 
+  /**
+   * Files extensions that need to be managed by Zinc together with class files
+   */
+  protected def auxiliaryClassFileExtensions: T[Seq[String]] = T {
```

Comment:
I think we should declare it in `JavaModule`, since we also use Zinc for Java-compilation. Although I currently can't tell if there is a need, wince we use zinc, we in fact run incremental Java compilation, and in it's nature we need a way to control auxiliary files removal.

Reply:
Done ✅